### PR TITLE
Add basic validation for the Individual Events duration field

### PR DESCRIPTION
**Pull request by fvzzy**
Addresses https://github.com/zendesk/volunteer_portal/issues/36

### CODE REVIEW

**Review by danielbreves — APPROVED** [review]

File: app/javascript/pages/IndividualEvents/index.js [hunk 1]
```diff
@@ -97,6 +97,9 @@ const validate = values => {
   if (!values.duration) {
     errors.duration = 'is required'
   }
+  if (values.duration > 8 * 60) {
```
Comment (craig-day) [inline app/javascript/pages/IndividualEvents/index.js]: where did the 8 hours come from? Before we merge this I would like to confirm with everyone that 8 hours is the max.

**Review by craig-day — CHANGES_REQUESTED** [review]
please also add a server side validation for this

**Comment by xuebingli** [comment]
@craig-day feedback addressed 💇‍♂️

File: app/javascript/pages/IndividualEvents/index.js (hunk 1 above)
Comment (xuebingli) [reply app/javascript/pages/IndividualEvents/index.js]: @azaiszen from CSR Committee suggests that 24 hours is a reasonable limit.

File: app/models/individual_event.rb [hunk 2]
```diff
@@ -11,7 +11,10 @@ class IndividualEvent < ApplicationRecord
   belongs_to :office
 
   validates :user, :organization, :event_type, :office,
-    :description, :date, :duration, presence: true
+            :description, :date, presence: true
+  validates :duration, numericality: { only_integer: true,
+                                       greater_than: 0,
+                                       less_than_or_equal_to: 1440 }
```
Comment (craig-day) [inline app/models/individual_event.rb]: lets take advantage of rails here and make this more readable

```rb
less_than_or_equal_to: 24.hours
```

**Review by craig-day — APPROVED** [review]
Looks good after the time helper addition

File: app/models/individual_event.rb (hunk 2 above)
Comment (xuebingli) [reply app/models/individual_event.rb]: I'm afraid this doesn't work as expected. `duration` on `IndividualEvent` is an integer representing minutes.

```
irb(main):018:0> 61 < 1.hour
=> true
```

Comment (craig-day) [reply app/models/individual_event.rb]: Okay

**Pull request merged**